modules/ROOT/pages/shared-bucket-access.adoc

```diff
@@ -1,18 +1,17 @@
 = Mobile-Web Data Sync
 :url-downloads: https://www.couchbase.com/downloads
 
-Starting in Sync Gateway 1.5, you can seamlessly extend an existing Couchbase Server deployment to connect with remote edge devices that are occasionally disconnected or connected.
+Shared Bucket Access enables document changes in a Couchbase Server bucket to be made through Couchbase SDKs, N1QL queries or Sync Gateway and vice versa.
+The following diagram shows different clients (mobile, web, desktop) writing data to the same bucket in Couchbase Server.
+This feature is available since Sync Gateway 1.5 and Couchbase Server 5.0.
+
```

Comment:
you are missing the key word "synchronized".  Shared bucket access allows server side and client side changes to be automatically synchronized

Reply:
Updated. I had missed the importance of that aspect.
Dropped the keyword "enabled" in the context of Couchbase Lite.